src/metaprob/autotrace.cljc

```diff
@@ -9,8 +9,8 @@
   [expressions stack]
   (map-indexed #(autotrace-expression %2 (cons %1 stack)) expressions))
 
-(defmacro autotrace [gen-expr]
-  (let [expr (expander/mp-expand gen-expr)
+(defmacro autotrace #?(:clj [gen-expr] :cljs [env gen-expr])
```

Comment:
I'm a bit confused by this. How is `autotrace` invoked in ClojureScript? It's a macro -- does it not already have access to some `env` info?